libs/core/src/lib/select/select.component.ts

```diff
@@ -74,9 +86,17 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnInit, OnD
     @Input()
     isOpen: boolean = false;
```

Comment:
Is there way, to control open state only programatically, so clicking inside won't trigger this flag?

Reply:
No there is not. I was also wondering what is the correct behavior for such scenario, but in my opinion isOpen should always reflect select open state.